src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfo.cs

```diff
@@ -38,9 +39,9 @@ internal abstract class JsonPropertyInfo
         public bool ShouldSerialize { get; private set; }
         public bool ShouldDeserialize { get; private set; }
 
+        public bool IsPropertyPolicy {get; protected set;}
```

Comment:
What does this property mean?

Reply:
The "PropertyPolicy" is documented elsewhere. It is internal of course. It is just re-using the JsonPropertyInfo mechanism to represent a pseudo-property that helps with validation and converters for types that don't have properties such a List or value type.

That said, I think we can think of a better name (outside of this PR)